go/vt/binlog/binlogplayer/binlog_player.go

```diff
@@ -171,20 +173,21 @@ func NewStats() *Stats {
 	bps.ReplicationLagSeconds.Store(math.MaxInt64)
 	bps.PhaseTimings = stats.NewTimings("", "", "Phase")
 	bps.QueryTimings = stats.NewTimings("", "", "Phase")
-	bps.QueryCount = stats.NewCountersWithSingleLabel("", "", "Phase", "")
-	bps.BulkQueryCount = stats.NewCountersWithSingleLabel("", "", "Statement", "")
-	bps.TrxQueryBatchCount = stats.NewCountersWithSingleLabel("", "", "Statement", "")
+	bps.QueryCount = stats.NewCountersWithSingleLabel("", "", "Phase")
+	bps.BulkQueryCount = stats.NewCountersWithSingleLabel("", "", "Statement")
+	bps.TrxQueryBatchCount = stats.NewCountersWithSingleLabel("", "", "Statement")
 	bps.CopyRowCount = stats.NewCounter("", "")
 	bps.CopyLoopCount = stats.NewCounter("", "")
 	bps.ErrorCounts = stats.NewCountersWithMultiLabels("", "", []string{"type"})
-	bps.NoopQueryCount = stats.NewCountersWithSingleLabel("", "", "Statement", "")
+	bps.NoopQueryCount = stats.NewCountersWithSingleLabel("", "", "Statement")
 	bps.VReplicationLags = stats.NewTimings("", "", "")
 	bps.VReplicationLagRates = stats.NewRates("", bps.VReplicationLags, 15*60/5, 5*time.Second)
-	bps.TableCopyRowCounts = stats.NewCountersWithSingleLabel("", "", "Table", "")
+	bps.TableCopyRowCounts = stats.NewCountersWithSingleLabel("", "", "Table")
 	bps.TableCopyTimings = stats.NewTimings("", "", "Table")
 	bps.PartialQueryCacheSize = stats.NewCountersWithMultiLabels("", "", []string{"type"})
 	bps.PartialQueryCount = stats.NewCountersWithMultiLabels("", "", []string{"type"})
 	bps.ThrottledCounts = stats.NewCountersWithMultiLabels("", "", []string{"throttler", "component"})
+	bps.DDLEventActions = stats.NewCountersWithSingleLabel("", "", "action")
```

Comment:
Any idea whether these tags are case-sensitive? I mean when filtering using PromQL. In this case, there is only 1 tag, but for multi-label counters, you might want to filter that way.

Reply:
For debug vars it's a big JSON document and it's likely case sensitive (technically depends on the client):
```
❯ curl -s localhost:15102/debug/vars | jq '.VReplicationThrottledCountTotal'
0
❯ curl -s localhost:15102/debug/vars | jq '.VReplicationThrottledCounttotal'
null
```

But the metrics are always lower-cased:
```
❯ curl -s localhost:15102/metrics | grep throttled
# HELP vttablet_v_replication_throttled_count_total The total number of times that vreplication has been throttled
# TYPE vttablet_v_replication_throttled_count_total counter
vttablet_v_replication_throttled_count_total 0
```

Reply:
Yes, I believe they are. For example this returns values in grafana with prometheus:
```
vttablet_errors{error_code="ABORTED"}
```

But this does not:
```
vttablet_errors{error_code="aborted"}
```